src/common/providers/https.ts

```diff
@@ -500,6 +502,53 @@ interface CallableTokenStatus {
   auth: TokenStatus;
 }
 
+function decodeToken(token: string): unknown {
+  const decoded = jwsDecode(token);
+  if (!decoded) {
+    return {};
+  }
+
+  let payload = decoded.payload;
+  if (typeof payload === 'string') {
+    try {
+      const obj = JSON.parse(decoded.payload);
+      if (typeof obj === 'object') {
+        payload = obj;
+      }
+    } catch (e) {}
+  }
+  return payload;
+}
+
+/**
+ * Decodes Auth ID token.
+ *
+ * This is exposed only for testing.
+ */
+/** @hidden */
```

Comment:
Let's use @internal to avoid letting other packages depend on this (in TypeScript at least..)

Reply:
Fixed!